components/forms/ltft/LtftFormView.tsx

```diff
@@ -177,7 +177,11 @@ export const LtftFormView = () => {
                       }
                       data-cy="BtnSubmit"
                     >
-                      {isSubmitting ? "Saving..." : "Submit"}
+                      {isSubmitting
```

Comment:
Code changes LGTM - although sonarcloud will probably shout at this nested ternary 😁